test/navigation/NavigatorForCorrectReturnSpec.scala

```diff
@@ -97,35 +97,35 @@ class NavigatorForCorrectReturnSpec extends SpecBase with DataHelper {
     "navigate to Check Your Answers page in CheckMode (when not new importer or new copacker)" in {
       val userAnswers = emptyUserAnswersForCorrectReturn
         .set(HowManyBroughtIntoUKPage, LitresInBands(1, 1)).success.value
-      val result = navigator.nextPage(HowManyBroughtIntoUKPage, CheckMode, userAnswers, Some(aSubscription))
+      val result = navigator.nextPage(HowManyBroughtIntoUKPage, CheckMode, userAnswers, None, Some(aSubscription))
```

Comment:
Use named parameters for each of these unit tests rather than passing None (goes for all of these) 👇